docs/gallery/SnowExhackweek.md

```diff
@@ -0,0 +1,18 @@
+# SnowEx Hackweek
+
+![SnowEx logo](https://snowex-hackweek.github.io/website/_static/logo.png)
+
+## Website
+
+## Projects
+
+## Outcomes
+
+## Evaluation
```

Comment:
definitely like including this and some testimonials, perhaps a basic description under the logo with key things like dates, number of participants